.github/workflows/verify.yaml

```diff
@@ -44,35 +44,34 @@ jobs:
         run: mvn -N io.takari:maven:wrapper -Dmaven=3.6.3
 
       - name: Build and Test All Variants
-        run: ./mvnw clean install -Pprod -Psql -Pkafkasql -Pmultitenancy -Dmaven.wagon.httpconnectionManager.maxTotal=30 -Dmaven.wagon.http.retryHandler.count=5
+        run: make SKIP_TESTS=false BUILD_FLAGS='-Dmaven.wagon.httpconnectionManager.maxTotal=30 -Dmaven.wagon.http.retryHandler.count=5' build-all
 
       - name: Login to DockerHub Registry
         if: github.event_name == 'push'
         run: echo ${{ secrets.DOCKERHUB_PASSWORD }} | docker login -u ${{ secrets.DOCKERHUB_USERNAME }} --password-stdin
 
-      - name: Build The Snapshot Docker Images
+      - name: Build The Images for Docker.io
         if: github.event_name == 'push'
-        run: ./.github/scripts/build-snapshot-images-for-docker.sh ${{ github.ref }}
+        run: ./.github/scripts/build-images.sh ${GITHUB_REF#refs/heads/} docker.io snapshot null
 
-      - name: List All The Tagged Docker Images
+      - name: List All The Images
         run: docker images
 
-      - name: Push The Snapshot Docker Images To Dockerhub
+      - name: Push The Images To Docker.io
         if: github.event_name == 'push'
-        run: ./.github/scripts/push-snapshot-images-to-docker.sh ${{ github.ref }}
+        run: ./.github/scripts/push-images.sh ${GITHUB_REF#refs/heads/} docker.io snapshot null
 
       - name: Login to Quay.io Registry
         if: github.event_name == 'push'
         run: docker login -u "${{ secrets.QUAY_USERNAME }}" -p "${{ secrets.QUAY_PASSWORD }}" quay.io
 
-      - name: Tag Snapshot Docker Images for Quay.io
-        if: github.event_name == 'push'
-        run: ./.github/scripts/tag-snapshot-images-for-quay.sh ${{ github.ref }}
-          
-      - name: List All The Tagged Docker Images
+      - name: Build The Images for Quay.io
         if: github.event_name == 'push'
+        run: ./.github/scripts/build-images.sh ${GITHUB_REF#refs/heads/} quay.io snapshot null
```

Comment:
overall looks really good, the only thing I'm wondering is if this will work fine. Is this tested? I'm not sure if it's correct to have the last parameter being explicitly "null" , if it works as expected is ok.

Reply:
I have tested it. The reason why I am passing `null` for the parameter `RELEASE_VERSION` is because we don't need it. Image tagged with the release-version is only built and pushed for concrete releases, not for snapshot ones.

Reply:
Maybe instead of passing `null` it might make sense to pass nothing at all?

Reply:
Looking at the impl of the script, it's not like the string `"null"` is needed.  You could pass `"foobar"` instead and it would still work.  So I would recommend just not passing anything.  Thoughts?

Reply:
+1

Reply:
Done